### ISSUE TITLE
Create a 3D's fingerprint  for the rotating display

### ISSUE THREAD

**Issue by eliselavy** [C2DH/journal-of-digital-history#532]
In order to do a presentation at DhD , the idea is to reuse the rotating display used during the 5 years's C2DH.
Is it possible to generate this for Tuesday morning?

A picture of the fingerprint need to be delivered.

**Comment by biumiamy** [comment]
Does it mean I need to create the fingerprint in an 3D environment? Or just a normal picture with the format size of the display?

**Comment by eliselavy** [comment]
@biumiamy yes coordinate with robert , i don't know the format required by this tool.
I will present the journal during the pause the 14 March
![Screenshot 2023-03-10 at 14 32 19](https://user-images.githubusercontent.com/35602279/224328918-4dcb872d-64f0-4746-8888-13595f9227a9.png)

**Comment by robertbeta** [comment]
@eliselavy, @biumiamy, I tested the 3D holographic projector ([Holovisio HV Show 78](https://www.holovisio.fr/holographic-propeller-hv-show-78/)) and it is very easy to use and install. You only need to download the software from [here](https://drive.google.com/drive/folders/16nyD4Z9ptfICs4RaLDXdQDpU5je_xYdR) and run it. I will come back with a step-by-step procedure.

I placed it in the DH Lab (MSH, 1st floor) to get it faster tomorrow.
